cognite/client/data_classes/assets.py

```diff
@@ -103,8 +100,7 @@ def _load(cls, resource: Union[Dict, str], cognite_client=None):
         if isinstance(resource, Dict):
             if instance.aggregates is not None:
                 instance.aggregates = AggregateResultItem(**instance.aggregates)
-        if instance.labels is not None:
-            instance.labels = [Label._load(label) for label in instance.labels]
+        instance.labels = Label._load_list(instance.labels)
```

Comment:
this seems a bit meh, either Label._load or LabelList._load?

Reply:
if LabelList is already a thing that is

Reply:
agree it does not really belong perfectly in this place - initially tried putting it in utils._auxillary, but it caused a nasty circular dependency issue that I wasn't able to sort out :cry: